src/BookStore/Domain/Model/Book.php

```diff
@@ -17,31 +17,70 @@
 class Book
 {
     #[ORM\Embedded(columnPrefix: false)]
-    public BookId $id;
+    public readonly BookId $id;
```

Comment:
Any blocker to make it private?

Reply:
Not really, but I thought maybe you guys prefer it this way, as the property is now read-only anyway.

Reply:
Fair :) I'd prefer some public method for consistency's sake

Reply:
Done